portal/views/patient.py

```diff
@@ -24,20 +26,27 @@
 @patient_api.route('/api/patient/')
 @oauth.require_oauth()
 def patient_search():
-    """Looks up patient from given parameters, returns FHIR Patient if found
+    """Looks up patient(s) from given parameters, returns FHIR Patient bundle
 
-    Takes key=value pairs to look up.  At this time, only email is supported.
+    Takes key=value pairs to look up.  Email, identifier searches supported.
 
-    Example search:
+    Example searches:
         /api/patient?email=username@server.com
+        /api/patient?identifier=http://us.truenth.org/identity-codes/external-study-id|123-45-678
 
-    Returns a FHIR patient resource (http://www.hl7.org/fhir/patient.html)
-    formatted in JSON if a match is found, 404 otherwise.
+    Identifier search pattern:
+        ?identifier=<system>|<value>
```

Comment:
Why not pass `system` and `value` as individual querystring parameters (eg `id_system`, `id_value`)? Seems error-prone (better than as JSON-in-URL though ;)

Reply:
FHIR compliance.  See https://www.hl7.org/fhir/DSTU2/search.html#token